Documentation/subcommands/prepare.md

```diff
@@ -40,7 +40,7 @@ c9fad0e6-8236-4fc2-ad17-55d0a4c7d742
 | `--no-overlay` |  `false` | `true` or `false` | Disable overlay filesystem |
 | `--no-store` |  `false` | `true` or `false` | Fetch images, ignoring the local store. See [image fetching behavior](../image-fetching-behavior.md) |
 | `--pod-manifest` |  `` | A path | The path to the pod manifest. If it's non-empty, then only `--net`, `--no-overlay` and `--interactive` will have effect |
-| `--port` |  `` | A port number | Ports to expose on the host (requires `--net`) |
+| `--port` |  `` | A port number | Ports to expose on the host (requires contained network). Syntax: --port=NAME:HOSTPORT |
```

Comment:
"contained network" could be a link to:

https://github.com/coreos/rkt/blob/master/Documentation/networking.md#contained-mode